network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_author_detail_header.html

```diff
@@ -0,0 +1,49 @@
+{% load l10n i18n localization static blog_tags wagtailimages_tags wagtailroutablepage_tags wagtailcore_tags %}
+
+{% get_root_or_page as blog_index_page %}
+
+<div class="tw-grid tw-grid-cols-12 large:tw-gap-40">
+    <div class="tw-col-span-12 large:tw-col-span-8">
+        <div class="tw-text-sm tw-text-gray-40">
+            <a class="tw-text-gray-40" href="{% pageurl blog_index_page %}">Blog</a>
+            <span> › </span>
+        </div>
+        <div class="tw-gap-8 tw-pt-10 medium:tw-inline-flex medium:tw-pt-7">
+            <div id="image-container" class="medium:tw-order-last">
+                <div class="tw-flex-shrink-0">
+                    {% with profile_image_classes="tw-rounded-full tw-w-28 tw-h-28" %}
+                        {% if author.image %}
+                            {% image author.image fill-120x120 as img_1x %}
+                            {% image author.image fill-240x240 as img_2x %}
+                            <img srcset="{{ img_1x.url }}, {{ img_2x.url }} 2x" src="{{ img_2x.url }}" width="{{ img_2x.width|unlocalize }}" height="{{ img_2x.height|unlocalize }}"  alt="{{ img_2x.alt }}" class="{{ profile_image_classes }}" />
+                        {% else %}
+                            <img src="{% static '_images/author-placeholder.jpg' %}" width="58px" height="58px" alt="{{ author.name }}" class="{{ profile_image_classes }}" />
+                        {% endif %}
+                    {% endwith %}
+                </div>
+            </div>
+            <h1 class="tw-mb-8">{{ author.name }}</h1>
+        </div>
+        <p class="tw-text-xl tw-pb-16">
+            {{ author.localized.introduction }}
+        </p>
+        {% if frequent_topics %}
+            <p class="tw-text-lg tw-text-gray-40 tw-uppercase">{% trans "Frequent Topics" %}</p>
+            <div class="tw-inline-flex tw-gap-6 tw-flex-wrap">
+                {% with link_classes="tw-text-black tw-px-8 tw-border-black tw-border tw-rounded-full tw-text-lg tw-font-normal tw-whitespace-nowrap tw-block tw-transition-colors" link_hover_classes="hover:tw-bg-black hover:tw-text-white hover:tw-no-underline hover:tw-border-white" %}
+                    {% for topic in frequent_topics %}
+                        {% localized_version topic as localized_topic %}
```

Comment:
Do we need this line? The topics will be localized at the `get_authors_frequent_topics` method, using `localize_queryset`...

Reply:
Great point! Updated